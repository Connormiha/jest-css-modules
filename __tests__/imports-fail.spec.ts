import style from './source/style-with-not.css';

describe('Style with :not selector', () => {
  it('should import default', () => {
    expect(style).toMatchSnapshot();
  });
});
