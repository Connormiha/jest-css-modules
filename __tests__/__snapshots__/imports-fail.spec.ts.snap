// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Style with :not selector should import default 1`] = `
Object {
  "class1": "class1",
  "class2": "class2",
  "class3": "class3",
}
`;
